modules/ocf/templates/kubectl/kubectl.conf.erb

```diff
@@ -0,0 +1,20 @@
+users:
+- name: ocfstaffer
```

Comment:
is the "-er" necessary?

Reply:
Actually I'll change it to kerberos-login since that better describes what it's doing anyways